lib/tasks/yarn.rake

```diff
@@ -0,0 +1,13 @@
+task :spec => ["yarn"]
+
+desc 'Installing dependencies with Yarn'
+task :yarn do
+    sh 'yarn install', verbose: false do | ok, _ |
+        if ok
+            puts '✅ Yarn install ran OK'.green
+        else
+            puts '⚠️ Failed to run yarn successfully, Stopping here'.red
```

Comment:
instead of `, Stopping here` I would say something more useful like `. Is Yarn installed?`